ml/src/dataset.py

```
@@ -0,0 +1,127 @@
+    if '거래날짜' in merged.columns:
+        merged['월'] = pd.to_datetime(merged['거래날짜']).dt.month.astype(str).str.zfill(2)
+
+    # 3. Tax 병합 on '제품카테고리'
+    if '제품카테고리' in merged.columns and 'Tax' in dfs:
+        merged = pd.merge(merged, dfs['Tax'], how='left', on='제품카테고리')
+    else:
+        raise KeyError("'제품카테고리'가 병합 중간에 없습니다.")
+
+    # 4. Discount 병합 on '월'
+    if '월' in merged.columns and 'Discount' in dfs:
+        discount_df = dfs['Discount'].drop(columns=['제품카테고리'])
```

Ruff (0.12.2)

Local variable `discount_df` is assigned to but never used

Remove assignment to unused variable `discount_df`

(F841)